dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScopeTest.kt

```diff
@@ -514,23 +518,27 @@ internal class RumViewManagerScopeTest {
     // region AppLaunch View
 
     @Test
-    fun `𝕄 start an AppLaunch ViewScope 𝕎 handleEvent { app not displayed, event is relevant }`(
+    fun `𝕄 start an AppLaunch ViewScope 𝕎 handleEvent { app not displayed, any event }`(
         forge: Forge
     ) {
         // Given
         CoreFeature.processImportance = RunningAppProcessInfo.IMPORTANCE_FOREGROUND
         testedScope.applicationDisplayed = false
-        val fakeEvent = forge.validAppLaunchEvent()
+        val fakeEvent = forge.anyRumEvent()
+        val appStartTimeNs = fakeEvent.eventTime.nanoTime - TimeUnit.MILLISECONDS.toNanos(500)
+        whenever(mockAppStartTimeProvider.appStartTimeNs) doReturn appStartTimeNs
 
         // When
         testedScope.handleEvent(fakeEvent, mockWriter)
 
         // Then
-        assertThat(testedScope.childrenScopes).hasSize(1)
+        val appStartTimeMs = TimeUnit.NANOSECONDS.toMillis(appStartTimeNs)
+        val scopes = if (fakeEvent is RumRawEvent.StartView) 2 else 1
+        assertThat(testedScope.childrenScopes).hasSize(scopes)
```

Comment:
minor suggestion (just to distinguish from `scopes` as collection)

```suggestion
        val scopesCount = if (fakeEvent is RumRawEvent.StartView) 2 else 1
        assertThat(testedScope.childrenScopes).hasSize(scopesCount)
```